src/opencl_rawsha256_fmt_plug.c

```diff
@@ -165,6 +165,9 @@ static void create_clobj(size_t gws, struct fmt_main *self)
 	uint32_t hash_id_size;
 	size_t mask_cand = 1, mask_gws = 1;
 
+	// Avoid memory leaks.
+	release_clobj();
+
```

Comment:
The above probably needs to be added to all other (not yours) formats as well, right?

Reply:
Yes, using bash-fu (if possible). I use a fancy IDE.

Reply:
What IDE, Eclipse?

Reply:
Visual Studio Code